third_party/grpc/generate_grpc_gyp

```diff
@@ -13,10 +13,10 @@ function gypify_file_list {
 
 cd src/third_party/grpc
 
-GPR_DIRS='src/src/core/profiling src/src/core/support'
-CORE_EXCLUDE_DIRS='src/src/core/statistics src/src/test'
+GPR_DIRS='src/src/core/lib/profiling src/src/core/lib/support src/src/core/ext/census'
+CORE_EXCLUDE_DIRS='src/src/test/core/statistics src/src/test'
 
-gpr_sources=$(find $GPR_DIRS -type f -name '*.c' | gypify_file_list)
+gpr_sources=$(find $GPR_DIRS -type f -name '*.c' ! -name 'window_stats.c' | gypify_file_list)
```

Comment:
it contains a definition that doesn't exist, I'll try excluding that entire dir.